ai/data/beelzex_deck.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "Beelzemon X Antibody",
+    "color": "Purple",
```

Comment:
Is this up-to-date with the current models?

Reply:
Great catch.
Works fine now.
Will have to delete the bot user online for this to take effect but it's very easy.
![image](https://github.com/WE-Kaito/digimon-tcg-simulator/assets/6946535/5d9bcbf2-3374-4ee4-b69c-1821c6b8d495)